pkg/crc/api/listener_nonwin.go

```diff
@@ -0,0 +1,18 @@
+// +build !windows
+
+package api
+
+import (
+	"net"
+
+	"github.com/code-ready/crc/pkg/crc/logging"
+)
+
+func createIPCListener(socketPath string) (net.Listener, error) {
+	listener, err := net.Listen("unix", socketPath)
```

Comment:
We can also check the socketPath length. For Darwin, the maximum is 104 bytes. For Linux, it's 108. (ref https://github.com/moby/vpnkit/pull/506 or https://github.com/moby/moby/pull/13408). 

But we can do that in an other PR!